system_test/helpers/aest_nodes.erl

```diff
@@ -148,6 +151,12 @@ stop_node(NodeName, Timeout, Ctx) ->
 kill_node(NodeName, Ctx) ->
     call(ctx2pid(Ctx), {kill_node, NodeName}).
 
+extract_archive(NodeName, Path, Archive, Ctx) ->
+    call(ctx2pid(Ctx), {extract_archive, NodeName, Path, Archive}).
+
+run_cmd_in_node_dir(NodeName, Cmd, Ctx) ->
```

Comment:
I am adding an `exec` function that uses the docker API, is there any specific things this is doing I may be missing?

Reply:
stdout